docs/guides/02_page_layer_widgets.md

```diff
@@ -139,6 +141,7 @@ Summary:
 - To create a layer you need to define a function that returns a deck.gl layer.
 - The layer must be added to the application layers array.
 - You need to add the source and the layer to the store.
+- `DEFAULT_PROPS` are required.
```

Comment:
... if filtering is required. There could be use cases of layers without it... , right?

Reply:
For client-side widgets filtering is required.. We can specify this in docs.